ReadMe.md

```diff
@@ -0,0 +1,215 @@
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 Math Magician App <a name="about-project"></a>
+
+> "Bookstore" is a website for book lovers. It is a Single Page App (SPA) that allows users to:
+
+- Make simple book addtion to their lists.
+- You can edit, remove and add.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- CSS
+- Javascript
+- React.js
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>React</summary>
+  <ul>
+    <li><a href="https://reactjs.org/docs/create-a-new-react-app.html">React</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **This is a SPA.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://the-normal-one1.github.io/bookstore/)
+
+## 🚀 Project Details
+
+- Solo Project
+  - [Ermiyas Demsew](https://github.com/The-Normal-One1)
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running follow these simple example steps.
+
+## Clone Git Repository
+
+-Reach to following Github URL:
+
+```
+https://github.com/The-Normal-One1/bookstore
+
+```
+
+-Copy the URL.
+
+-In Visual Studio Code “Terminal” change the current working directory to the location where you want the cloned directory.
+
+-Type git clone in the terminal, paste the URL you copied earlier, and press “enter” to create your local clone.
+
+```
+$ git clone {repository URL}
+```
+
+# Install Dependencies(with following Command)
+
+- npm install
+
+# Build webpack <a name="setup"></a>
+
+- run command
+
+```
+npm run build
+```
+
+## Start server
+
+```
+npm start
+```
+
+### Prerequisites <a name="prerequisites"></a>
+
+In order to run this project you need:
+
+- [x] Basic Knowledge of **HTML**, **CSS**, & **JavaScript**.
+- [x] Basic knowledge of **ES6** syntax.
+- [x] Basic knowledge of **Git**, **GitHub** &**Gitflow**.
+- [x] Basic knowledge of **Linters**.
+- [x] Basic knowledge of **Webpack**.
+- [x] Basic knowledge of **VS Code** or any other code-editors.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+    _cd PATH_
+
+    _git clone  https://github.com/The-Normal-One1/bookstore.git
+
+_cd Javascript-Group-Capstone_
+
+_npm install_
+
+_npm run build_
+
+_npm start_
+
+### Install
+
+Install this project with:
+
+```sh
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npm test
+```
+
+### Deployment <a name="triangular_flag_on_post-deployment"></a>
+
+You can deploy this project using:
+
+```sh
+  npm start
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+# Authors
+Amk lass
+
+GitHub:[@amklass11](https://github.com/amklass11)
+LinkedIn:[@AmkLAss](https://www.linkedin.com/in/amk-lass-521565196/)
```

Comment:
[OPTIONAL] I think you should add more links as this will increase the chances of you. being reached easily